pom.xml

```diff
@@ -1918,7 +1902,8 @@
                                         <!-- Exclude javax dependencies in favor of Jakarta -->
                                         <exclude>javax.activation:activation</exclude>
                                         <exclude>javax.activation:javax.activation-api</exclude>
-                                        <exclude>javax.annotation:javax.annotation-api</exclude>
+                                        <!-- Uncomment when the dependency is removed from io.fabric8.kubernetes-client -->
+                                        <!-- <exclude>javax.annotation:javax.annotation-api</exclude>-->
```

Comment:
@vinokurig could you clarify this part?

Reply:
If maven meets a dependency from the banned list, the build fails. The new version of kubernetes client brings the banned dependency as a nested dependency. To avoid the build failure I temporary removed the nested dependency from the list of banned dependencies until the fabric8 kubernetes client removes it, see: https://github.com/fabric8io/kubernetes-client/blob/main/kubernetes-model-generator/pom.xml#L112-L114